content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,140 +2,147 @@
 the ticket is moved to the :guilabel:`Reach Stage`.
 
 .. image:: sla/sla-passing-failing.png
    :align: center
-   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk
+   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk.
 
 Analyzing SLA performance
 =========================
 
 The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
-Agreement)` is fulfilled, as well as the success rate of individual policies. Navigate to the report
-and corresponding pivot table by going to :menuselection:`Helpdesk --> Reporting --> SLA Status
-Analysis`.
+Agreement)` is fulfilled, as well as the performance of individual team members. Navigate to the
+report and corresponding pivot table by going to :menuselection:`Helpdesk app --> Reporting --> SLA
+Status Analysis`.
 
 Using the Pivot view
 --------------------
 
-By default, the report is displayed in a :guilabel:`Pivot` view, and is filtered to show the number
-of SLAs failed and the failure rate over the last 30 days, grouped by team.
+By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
+Agreement)` policies in the database with tickets that are in progress, failed to fulfill a policy,
+or have satisfied a policy are listed. By default, they are grouped by team, and ticket count.
 
 .. image:: sla/sla-status-analysis.png
    :align: center
-   :alt: View of the SLA status analysis report in Odoo Helpdesk
+   :alt: View of the SLA status analysis report in Odoo Helpdesk.
+
+To change the display, or to add additional measurements, click the :guilabel:`Measures` button to
+reveal a drop-down menu of reporting criteria, and choose from the options available. Whenever a
+measurement is picked, a check mark will appear in the drop-down menu to indicate that that
+measurement is included, and a corresponding new column will emerge in the pivot table to show the
+relevant calculations.
 
-To add the number of SLAs passed or in progress, click the :guilabel:`Measures` button to reveal a
-drop-down menu of reporting criteria, and choose from the options available based on the
-measurements preferred. Whenever a measurement is picked, a check mark will appear in the drop-down
-menu to indicate that that measurement is included, and a corresponding new column will emerge in
-the pivot table to show the relevant calculations.
+.. image:: sla/sla-pivot-measures.png
+   :align: center
+   :alt: View of the available measures in the SLA status analysis report.
 
-To add a group to a row or column, click the plus :guilabel:` + ` button next to
-:guilabel:`Total`, and then select one of the groups. To remove one, click the minus :guilabel:` - `
-button and deselect.
+To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to :guilabel:`Total`,
+and then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon and
+deselect.
```

Comment:
I don't see a + next to the total, it shows me a -. I do have a + next to '2 days to start', etc. Maybe just mention in generics that you click a + to add, and a - to remove? This is what I see:
![image](https://github.com/odoo/documentation/assets/121518652/bca3b4ff-a54c-4dc4-a440-5b4704d45cfe)